decidim-admin/app/controllers/decidim/admin/dashboard_controller.rb

```diff
@@ -12,7 +12,7 @@ def show
 
       private
 
-      def policy_class
+      def policy_class(_record)
```

Comment:
Yes, I left it on purpose. I think we will end up in a point where we will need to use different policies in a single action. 

Anyway, it can be removed.